docs/src/examples/components/Button/Types/ButtonExample.shorthand.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react'
```

Comment:
these changes are introduced for the sake of example to reviewer - won't be merged